File: src/main/java/yunas/server/EmbeddedServerFactory.java
package yunas.server;

import org.eclipse.jetty.server.Server;
import yunas.configuration.Configuration;

/**
 * Server Factory For Yunas.
 *
 * @author Yosuke Kobayashi
 */
public interface EmbeddedServerFactory {

    public Server create(Configuration conf);
}

File: src/main/kotlin/org/yunas/server/EmbeddedServerFactory.kt
package org.yunas.server

import org.yunas.configuration.Configuration
import org.eclipse.jetty.server.Server

/**
 * Server Factory For Yunas.
 *
 * @author Yosuke Kobayashi
 */
interface EmbeddedServerFactory {

    fun create(conf: Configuration): Server
}
